File: Solutions/0171/0171.cpp
// Author: washing
// DateTime: 2021/1/27 13:36
// File: 0171.cpp
// Desc: 

class Solution {
public:
    int titleToNumber(string s) {
        int n;
        int num = 0;
        for (auto &c : s){
            n = (int)c;
            num = num * 26 + (n-64);
        }

        return num;
    }
};

File: Solutions/0171/0171.java
// Author: washing
// DateTime: 2021/1/27 10:34
// File: 0171.java
// Desc: 

class Solution {
    public int titleToNumber(String s) {
        char i;
        int n=0;
        for (int idx=0; idx < s.length(); idx++){
            i = s.charAt(idx);
            n = n*26+(int)i-64;
        }
        return n;
    }
}